recipes/libgit2/all/FindLIBSSH2.cmake

```diff
@@ -0,0 +1,51 @@
+#
```

Comment:
This file shouldn't be neccessary

Reply:
See https://github.com/conan-io/conan-center-index/pull/286#discussion_r342316021